recipes/traviscli/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "traviscli" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/traviscli-{{ version }}.tar.gz
+  sha256: b93a5e7803874e0a963b8b7b085b54dcd6a826b6662732f8dff5c3b7b0b29c17
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - traviscli=travisci.TravisCli:commandHandler
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - click
+    - python >=3.6
+    - pytravisci
+
+test:
+  imports:
+    - travisci
+    - travisci.exceptions
+  commands:
```

Comment:
Can you also check that the entry point is working:

```
commands:
   - pip check
   - traviscli --help
```

Reply:
yup I'll fix this on the feedstock

Reply:
Any reason not to just include it here before the merge?

Reply:
Argh because I had a brainfart and I saw that close better and thought this was already merged. I can fix this now